models/fixtures/training/bar.json

```diff
@@ -0,0 +1,93 @@
+{
+  "training": [
+    {
+      "role": "Bar",
+      "pass_auto": true,
+      "pass_mark": 100,
+      "url": "",
+      "questions": [
+        {
+          "question": "A proof of age document is reliable if it has",
+          "answers": [
+            {
+              "answer": "Local authority logo",
+              "correct": false
+            },
+            {
+              "answer": "PASS hologram",
+              "correct": true
+            },
+            {
+              "answer": "Police authority stamp",
+              "correct": false
+            }
+          ]
+        },
+        {
+          "question": "What is the on-the-spot fine for selling alcohol to someone who is under 18?",
```

Comment:
To help review (current and future) and maintenance, can we include some supporting references for each question?  (perhaps via a hyperlink or two in a commit message)

Reply:
(update: I don't think we need a reference for each individual question, if they're all sourced from the same place.  the [original bar training document](https://github.com/emfcamp/Website/pull/920#discussion_r575707621) would be fine)

Reply:
(...I think.  just checking what the retention policy for IRCCloud CDN documents is...)

Reply:
@jayaddison This is short term so next steps after this is likely migration of that content to something within the website, at which point the referencing from URL will hopefully make more sense.

Reply:
Starting to think that perhaps I should rename the file to be bar_test.json and then we can have bar_content.json or similar. That sound like a sensible plan?

Reply:
I'll be reviewing and updating the training ahead of the 2022 event. I would like the training material to be hosted on this website; when it is, this repo can become the original document.

Reply:
(uncertain so far; but durable enough for the emfcamp website, I reckon)